tools/scalar-schema/src/scalar_schema/core.clj

```diff
@@ -8,6 +8,8 @@
   [[nil "--cassandra" "Operate for Cassandra"]
    [nil "--cosmos" "Operate for Cosmos DB"]
    ["-h" "--host DB_HOST" "Address of Cassandra like IP or URI address of your Cosmos DB account"]
+   ["-P" "--port DB_PORT" "Port of Cassandra. This option is ignored when Cosmos DB."
+    :default 9042 :parse-fn #(Integer/parseInt %)]
```

Comment:
I think this port number might be used for other data stores in the future.
So, the default value should be set in the `cassandra_schema.clj` as I've suggested.

Reply:
Should `:default 9042` here be removed?

Reply:
Yes, I think so. Each data store's schema process couldn't set each default value if the default value would be set here.